packages/tempo/doc/tempo.ticker.md

```
@@ -18,7 +18,7 @@ Tempo.extend(TickerModule);
 ```
 
 ### Direct Access
-If you need to access the [Reporting & Registry](#reporting--registry) API (like `Ticker.active`), you should import the `Ticker` namespace:
+If you need to access the [Reporting & Registry](#reporting-registry) API (like `Ticker.active`), you should import the `Ticker` namespace:
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)